test/optimization/test_slsqp.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
```

Comment:
I think copyright should ideally just be 2020 since this a brand new file

Reply:
Updated